### PR TITLE
Fix bug rename

### PR DESCRIPTION
**Pull request by MioRtia**
- Fix bug: when renaming a file, the renamed file is selected at the end of the operation
- When renaming an open file, the tab name is changed too

### CODE REVIEW

File: gse-app/src/main/java/com/powsybl/gse/app/ProjectPane.java [hunk 1]
```diff
@@ -585,17 +599,33 @@ private MenuItem createDeleteProjectNodeItem(List<? extends TreeItem<Object>> se
...
+                                                .stream()
+                                                .filter(item -> item.getValue() instanceof ProjectNode &&
+                                                        ((ProjectNode) item.getValue()).getId().equals(selectedTreeNode.getId()))
+                                                .findFirst()
+                                                .orElseThrow(NoSuchElementException::new));
+                                refreshTabs();
+                                change.getList().removeListener(this);
+                            }
+                        }
+                    });
```
Comment (mathbagu) [inline gse-app/src/main/java/com/powsybl/gse/app/ProjectPane.java]: This is a hack: you provide a listener that removes itself to keep the selected node. I think there is a better solution for this: change the value of the selected node, and refresh it should be sufficient

Comment (geofjamg) [reply gse-app/src/main/java/com/powsybl/gse/app/ProjectPane.java]: The selected node is detroyed by the refresh, so there is no way until refresh (which is ansynchronous) complete, to known what will be the new TreeItem instance.

Comment (geofjamg) [reply gse-app/src/main/java/com/powsybl/gse/app/ProjectPane.java]: Also there is also the same issue when we move an element. The tab view title contains le full path of the element and  is not refreshed

Comment (geofjamg) [reply gse-app/src/main/java/com/powsybl/gse/app/ProjectPane.java]: And also even when renaming it does not cover all cases.
Suppose you have 2 nested folders test1 and test2 and a case inside test2.
open the case, rename test1 and the tab title is not refreshed.

**Pull request merged**